cmd/blockchaincmd/change_weight.go

```diff
@@ -150,8 +148,8 @@ func setWeight(_ *cobra.Command, args []string) error {
 		BlockchainName: blockchainName,
 	}
 
-	if rpcURL == "" {
-		rpcURL, _, err = contract.GetBlockchainEndpoints(
+	if flags.RPC == "" {
+		flags.RPC, _, err = contract.GetBlockchainEndpoints(
```

Comment:
I'm not sure if changing input flags during code execution is the thing we want to do in the future, after the redesign